samples/snippets/src/main/java/com/example/storage/bucket/SetBucketAutoclass.java

```diff
@@ -21,32 +21,54 @@
 import com.google.cloud.storage.Bucket;
 import com.google.cloud.storage.BucketInfo.Autoclass;
 import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.Storage.BucketTargetOption;
+import com.google.cloud.storage.StorageClass;
 import com.google.cloud.storage.StorageOptions;
 
 public class SetBucketAutoclass {
-  public static void setBucketAutoclass(String projectId, String bucketName) {
+  public static void setBucketAutoclass(
+      String projectId, String bucketName, StorageClass storageClass) throws Exception {
     // The ID of your GCP project
     // String projectId = "your-project-id";
 
     // The ID of your GCS bucket
     // String bucketName = "your-unique-bucket-name";
 
-    // Whether to set Autoclass to on or off.
-    // Note: Only update requests that disable qutoclass are currently supported.
-    // To enable autoclass, you must enable it at bucket creation time.
+    // The storage class that objects in an Autoclass bucket eventually transition to if not read
+    // for a certain length of time
+    // StorageClass storageClass = StorageClass.NEARLINE;
```

Comment:
```suggestion
    // StorageClass storageClass = StorageClass.ARCHIVE;
```

Reply:
Apologies for the canonical sample changes, let's update this sample to set `enabled = true`  with `storageClass = StorageClass.ARCHIVE`. This is more aligned with the CUJ for setting terminal storage class. Thank you!